From 3b830edc8f5ee89f2b4e072237d7327b46d03317 Mon Sep 17 00:00:00 2001
From: Evgeny Vereshchagin <evvers@ya.ru>
Date: Tue, 10 Apr 2018 22:10:52 +0300
Subject: [PATCH] hwdb: drop a space (#8701)

This should make `parse_hwdb.py` and the CI happy again.

(cherry picked from commit cf45dd36282368d5cdf757cac2cf1fc2b562dab2)
---
 hwdb/60-evdev.hwdb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hwdb/60-evdev.hwdb b/hwdb/60-evdev.hwdb
index 26c7f3b02d..532a1cc59c 100644
--- a/hwdb/60-evdev.hwdb
+++ b/hwdb/60-evdev.hwdb
@@ -467,7 +467,7 @@ evdev:name:SynPS/2 Synaptics TouchPad:dmi:*svnTOSHIBA:pnSATELLITER830*
  EVDEV_ABS_01=1045:4826:76
  EVDEV_ABS_35=1238:5785:53
  EVDEV_ABS_36=1045:4826:76
- 
+
 #########################################
 # Razer
 #########################################
-- 
2.17.0

